CommonMark/Func.cs

```diff
@@ -5,6 +5,7 @@ namespace CommonMark
 #if v2_0
     /// <summary>An alternative to <c>System.Func</c> which is not present in .NET 2.0.</summary>
     public delegate TResult Func<in T, out TResult>(T arg);
+    public delegate TResult Func<in T1, in T2, out TResult>(T1 arg1, T2 arg2);
```

Comment:
please add the comment to the public declaration. can be copy-paste from the previous one, simply so that the compiler does not warn about missing comment